src/parse_entity_ACL.c

```diff
@@ -20,9 +20,10 @@ void	ft_parse_ambient(char *line, t_light *ambient)
 
 void	ft_parse_camera(char *line, t_cam *cam)
 {
-	ft_parse_float_block(&line, &cam->look_from);
-	ft_parse_float_block(&line, &cam->look_at);
+	ft_parse_float_block(&line, &cam->position);
```

Comment:
So the first parse into position is ok.
The second one can be parsed into a variable called direction, but then in the camera initiation, this direction must be added to position to arrive at look_at. Then with look_at and look_from(=position) we can derive the base vector w and the focal length. See previously posted code.

Reply:
Like above: I think we would just do some useless operations.

Reply:
Per further explanations above. If need we can talk in person next week.